.github/workflows/build.yml

```diff
@@ -59,32 +55,57 @@ jobs:
     needs:
       - get-go-version
       - get-product-version
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     strategy:
       matrix:
-        goos: ["linux"]
-        goarch: ["amd64", "arm64", "arm"]
-      fail-fast: true
-
+        goos: [linux]
+        goarch: ["arm64", "amd64"]
```

Comment:
It looks like we're no longer releasing ARM 32-bit. We should add that to the changelog

Reply:
I could also restore `arm`? I just looked at exec2 driver to refresh the gha versions etc., didn't notice I removed `arm` accidentally. fwiw Nomad is still released on 32-bit arm so why not podman driver. what do you think?

Reply:
> fwiw Nomad is still released on 32-bit arm

Oh, interesting. I thought we weren't anymore but that's 32-bit Intel. We closed https://github.com/hashicorp/nomad-enterprise/issues/1053 (internal) without ever addressing 32-bit ARM.

In the meantime, yeah it doesn't make sense to remove the build until we've removed the last 32-bit build for Nomad too.